src/utils/common/apiUtil.ts

```diff
@@ -1,3 +1,5 @@
 export const getEndpoint = () => {
-    return process.env.NEXT_PUBLIC_ENTE_ENDPOINT || "https://api.ente.io";
+    const endPoint = process.env.NEXT_PUBLIC_ENTE_ENDPOINT ?? "https://api.ente.io";
+    console.log(endPoint);
```

Comment:
Avoid  `console.log` these will reflect in production logs as well and are also not a very efficient way of debugging. 
I will add a lint check, which will get rid of this.